sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/EventHubsExporterIntegrationTest.java

```diff
@@ -69,6 +68,7 @@ public void producerTest() throws InterruptedException {
         assertTrue(exporterCountDown.await(5, TimeUnit.SECONDS));
     }
 
+    @Disabled
```

Comment:
is this disabled intentionally?

Reply:
Yes, processor tests run into an issue in CI due to partition contention. This test needs to be updated to use a unique consumer group each time it runs which is currently not configurable through ARM template. This needs more investigation and I have linked an issue to track this.